src/components/Tabs/Tabs.scss

```diff
@@ -3,13 +3,18 @@
 $item-min-height: rem(16px);
 $item-min-width: rem(50px);
 $item-vertical-padding: $item-min-height / 2;
+$border-bottom-height: border-width(thicker);
+$tab-separation-spacing: spacing(tight);
+$tab-item-margin-top: 0.1rem;
 $focus-state-box-shadow-color: rgba(92, 106, 196, 0.8);
 
 .Tabs {
   display: flex;
   flex-wrap: wrap;
   margin: 0;
-  padding: 0;
+  padding: var(--p-border-radius-wide, 0);
```

Comment:
Note:

This should be cleaned up so that we aren't using `border-radius-wide` and instead apply these styles within the `newDesignLanguage` class.

Reply:
Yes, I agree, this should just be using `spacing(tight)`. Using this custom property would be a little unpredictable.

Reply:
Let's get that change made, and use `spacing(base)`. No need to mis-use a token.